libexec/cli/list.help

```diff
@@ -0,0 +1,32 @@
+USAGE: singularity [...] list [list options...]
+
+List all running named instances
```

Comment:
I was actually thinking of using status to get more detailed information about a particular container instance. lol